### PR TITLE
Preferences polishing

### PR DESCRIPTION
**Pull request by ronso0**
* remove redundant analyzer combobox labels
* remove redundant rec directory label
* prevent incomplete skin description (somehow the last words may not show up)
* layout tweaks

### CODE REVIEW

File: src/preferences/dialog/dlgprefvinyldlg.ui [hunk 1]
```diff
@@ -543,6 +543,23 @@
...
+        </property>
+        <property name="sizeType">
+         <enum>QSizePolicy::MinimumExpanding</enum>
+        </property>
+        <!--property name="sizeHint" stdset="0">
+         <size>
+          <width>10</width>
+          <height>20</height>
+         </size>
+        </property-->
```
Comment (Be-ing) [inline src/preferences/dialog/dlgprefvinyldlg.ui:559]: Did you forget to remove this?

Comment (ronso0) [reply src/preferences/dialog/dlgprefvinyldlg.ui]: jup. already force-pushed

**Comment by Be-ing** [comment]
```
[7/604] Automatic MOC and UIC for target mixxx-lib
AutoUic: /home/be/sw/mixxx/src/preferences/dialog/dlgprefvinyldlg.ui: Warning: The name 'horizontalSpacer' (QSpacerItem) is already in use, defaulting to 'horizontalSpacer1'.
```

**Comment by Holzhaus** [comment]
> ```
> [7/604] Automatic MOC and UIC for target mixxx-lib
> AutoUic: /home/be/sw/mixxx/src/preferences/dialog/dlgprefvinyldlg.ui: Warning: The name 'horizontalSpacer' (QSpacerItem) is already in use, defaulting to 'horizontalSpacer1'.
> ```

Other than this, this PR looks good.

**Comment by JosepMaJAZ** [comment]
I am not sure if I understand it... Why is the recording directory redundant?

**Comment by Holzhaus** [comment]
> you simply removed the label because it is the same name than the group box that only contains this option.
> Yet, it is common to have a label in front of the field.

Yes. The issue is that we use labeled group boxes for single items. We could restore the label, rename the group box to "Recording Files" and move the "Create CUE File" and "Split files at X GB" into that group box.

**Comment by ronso0** [comment]
@Holzhaus that's what I thought last night, too, but I was already too lazy to push around boxes.
like this:
image

**Comment by Holzhaus** [comment]
@ronso0 exactly, that's what I meant. 👍

**Comment by ronso0** [comment]
oki, done.
the VC ui spacer warning is also fixed.

**Review by Holzhaus — APPROVED** [review]
Nice, thank you!

**Pull request merged**